README.md

```diff
@@ -63,6 +63,27 @@ To use the `couchbase` backend, setup a model using the [couchbase3 connector](h
 }
 ```
 
+To use a configurable loopback model as the ttl (only supported with couchbase currently) , omit the ttl option in the mixin configuration and use `lbEnvVar`, for example if we have a variable defined in our config as `cacheExpiryTTL` , we can use it as following within the mixin
```

Comment:
I don't think we need this extra implementation, you can override the ttl from the Model method if needed, or maybe just document this case in README?
```
module.exports = Message => {
  disableRemoteMethods(Message);

  Message.mixin('CacheModel', {
    backend: 'couchbase',
    ttl: app.get('msgBucketTTL')
  });

  // omit the remaining ...
  
```

Reply:
@xavierchow hmm I considered this too, reason for not picking it was too abstract implementation to the mixin, I can remove the extra keyword if you prefer this

Reply:
From the mixin's perspective, it cares about the TTL, it doesn't need to know if the TTL is a hardcoded number or an env var passed from the env var.
How to pass the TTL to the mixin is in the scope of the application and under the control of the application developer. I'd suggest removing the `lbEnvVar` usage.

Reply:
@xavierchow understood, but I think it's just another way to pass in a value my goal was to make it more flexible. I have removed it, please take another look, thanks